Client/qtTeamTalk/preferencesdlg.cpp

```diff
@@ -1,1877 +1,1980 @@
+        ui.ServnameChkBox->setChecked(ttSettings->value(SETTINGS_DISPLAY_SERVNAME,
+                                                     SETTINGS_DISPLAY_SERVNAME_DEFAULT).toBool());
+
+        ui.languageBox->clear();
+        ui.languageBox->addItem("");
+        QDir dir( TRANSLATE_FOLDER, "*.qm", QDir::Name, QDir::Files);
+        QStringList languages = dir.entryList();
+        for(int i=0;i<languages.size();i++)
+        {
+            QString name = languages[i].left(languages[i].size()-3);
+            ui.languageBox->addItem(name, name);
+        }
+        QString lang = ttSettings->value(SETTINGS_DISPLAY_LANGUAGE, "").toString();
+        int index = ui.languageBox->findData(lang);;
+        if(index>=0)
+            ui.languageBox->setCurrentIndex(index);
+    }
+    break;
+    case CONNECTION_TAB :  //connection
+    {
+        ui.autoconnectChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_AUTOCONNECT, false).toBool());
+        ui.reconnectChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_RECONNECT, true).toBool());
+        ui.autojoinChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_AUTOJOIN, true).toBool());
+        ui.maxpayloadChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_QUERYMAXPAYLOAD, false).toBool());
+#ifdef Q_OS_WIN32
+        QString appPath = QApplication::applicationFilePath();
+        appPath = QDir::toNativeSeparators(appPath);
+        ui.winfwChkBox->setChecked(TT_Firewall_AppExceptionExists(_W(appPath)));
+#endif
+        ui.subusermsgChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_SUBSCRIBE_USERMSG, true).toBool());
+        ui.subchanmsgChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_SUBSCRIBE_CHANNELMSG, true).toBool());
+        ui.subbcastmsgChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_SUBSCRIBE_BROADCASTMSG, true).toBool());
+        ui.subvoiceChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_SUBSCRIBE_VOICE, true).toBool());
+        ui.subvidcapChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_SUBSCRIBE_VIDEOCAPTURE, true).toBool());
+        ui.subdesktopChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_SUBSCRIBE_DESKTOP, true).toBool());
+        ui.submediafileChkBox->setChecked(ttSettings->value(SETTINGS_CONNECTION_SUBSCRIBE_MEDIAFILE, true).toBool());
+        ui.tcpportSpinBox->setValue(ttSettings->value(SETTINGS_CONNECTION_TCPPORT, 0).toInt());
+        ui.udpportSpinBox->setValue(ttSettings->value(SETTINGS_CONNECTION_UDPPORT, 0).toInt());
+    }
+    break;
+    case SOUND_TAB :  //sound system
+        initDevices();
+        ui.mediavsvoiceSlider->setValue(ttSettings->value(SETTINGS_SOUND_MEDIASTREAM_VOLUME,
+                                        SETTINGS_SOUND_MEDIASTREAM_VOLUME_DEFAULT).toInt());
+        break;
+    case SOUNDEVENTS_TAB :  //sound events
+    {
+        ui.spackBox->clear();
+        ui.spackBox->addItem("Default");
```

Comment:
Also tr("Default") here.

Reply:
I prefer let this not translated because it's a pack name, all users should have same default pack name I think.